packages/react-dev-utils/formatWebpackMessages.js

```diff
@@ -46,7 +46,7 @@ function formatMessage(message, isError) {
     if (threadLoaderIndex !== -1) {
       return;
     }
-    if (line.indexOf('from thread-loader (worker') !== -1) {
+    if (/thread.loader/i.test(line)) {
```

Comment:
I think it might be better to replace the `.` in the regular expression with `\s` to test for whitespace.

Reply:
sure, but what if they put back the dash?

Reply:
I see, that make sense.